### DISCUSSION TITLE
How to understand "reduce_zero_label"?

### DISCUSSION THREAD

**Discussion by WilliamJames0741** [open-mmlab/mmengine#1249] [Q&A]
I found that in the dataset "ade20k", the "reduce_zero_label" is set to "True", and the net like segnext is set to 150 classes(but include "background" there are 151 classes). However, when I use the well trained model to inference, its output has the "background" class. I wound why and how to understand the "reduce_zero_label" set? And how can I achieve the same functionality?

**Comment by zhouzaida** [comment]
Hi, you can continue the question in https://github.com/open-mmlab/mmsegmentation/discussions.

**Comment by WilliamJames0741** [comment]
谢谢，您的邮件我已收到！
